src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/filters/events/EventStatusFilter.kt

```diff
@@ -0,0 +1,58 @@
+/*******************************************************************************
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.exactpro.th2.rptdataprovider.entities.filters.events
+
+import com.exactpro.th2.rptdataprovider.entities.filters.Filter
+import com.exactpro.th2.rptdataprovider.entities.filters.info.FilterInfo
+import com.exactpro.th2.rptdataprovider.entities.filters.info.FilterParameterType
+import com.exactpro.th2.rptdataprovider.entities.filters.info.Parameter
+import com.exactpro.th2.rptdataprovider.entities.responses.Event
+import com.exactpro.th2.rptdataprovider.services.cradle.CradleService
+
+class EventStatusFilter(
+    requestMap: Map<String, List<String>>,
+    cradleService: CradleService
+) : Filter<Event>(requestMap, cradleService) {
+
+    private var status: Boolean = false
+    override var negative: Boolean = false
+
+    init {
+        negative = requestMap["${filterInfo.name}-negative"]?.first()?.toBoolean() ?: false
+        status = requestMap["${filterInfo.name}-values"]?.first()?.toBoolean() ?: false
```

Comment:
Maybe it would be better to create filters uniformly and throw an exception when -values is not specified?